script.js

```diff
@@ -1,4 +1,68 @@
 var main = function (input) {
```

Comment:
Please use ESLint when developing for formatting. Your formatting is non-standard and that makes it harder for everyone to read. This would not be accepted in industry.